### DISCUSSION TITLE
Widgets Not Displaying Data

### DISCUSSION THREAD

**Discussion by bxsteez** [Lissy93/dashy#1312] [Help and Support]
I have setup up the Pihole widget and the Proxmox widget and they show connected but there is no data.  I checked the developer options in my browser and there are no errors.  I ran the api request directly and the data is there.  I'm not sure what is going on??

![image](https://github.com/Lissy93/dashy/assets/56691147/99cfe4d5-3bf6-453d-a5f0-c6ecce1a4a9c)

**Comment by CrazyWolf13** [comment]
> I have setup up the Pihole widget and the Proxmox widget and they show connected but there is no data.  I checked the developer options in my browser and there are no errors.  I ran the api request directly and the data is there.  I'm not sure what is going on??
> 
> ![image](https://github.com/Lissy93/dashy/assets/56691147/99cfe4d5-3bf6-453d-a5f0-c6ecce1a4a9c)
> 

Regarding the pi-hole widget, did you specify the api-key? Since some updates pi-hole always requires an api-key.
